### PR TITLE
Test

### PR DESCRIPTION
**Pull request by InigoP**

### CODE REVIEW

File: tests/testthat/test-dataset.R [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    pivot_longer(cols = '1980':'2000',
+                 names_to ="year",
+                 values_to = 'delta_temperature')}) %>%
+  pivot_wider(id_cols = c('month','year'),
+              names_from = location,
+              values_from = delta_temperature) %>%
+    rename(n_dlt_delta_temperature = 'North Delta',
+           s_dlt_delta_temperature = 'South Delta')
+
+max_river_celsius <- 50
```
Comment (ErinCain) [inline tests/testthat/test-dataset.R:59]: Here you test that all fall below a max river Celsius. Would it be good to also test that they are all greater than a min?

File: R/data.R [hunk 2]
```diff
@@ -96,7 +96,7 @@
 #' Degree Days
 #'
 #' @description The monthly accumulated degree days °C
-#' @format a 3 dimensional array [31 watersheds, 12 months, 21 years]
+#' @format a 3 dimensional array [31 watersheds, 12 months, 22 years]
```
Comment (ErinCain) [inline R/data.R:99]: Good catch here. Ideally these would be in a different PR since it is documentation changes and not tests but fine to leave in for now.

File: tests/testthat/test-dataset.R (hunk 1 above)
Comment (InigoP) [reply tests/testthat/test-dataset.R:59]: min added!
